### PR TITLE
Automated Waystation

### PR DESCRIPTION
**Pull request by Stev-the-third**
Adds an Automated Waystation, an ASSN Department of Colonization sensor relay repurposed by the Coalition government to serve as a restpoint and refuelling station for weary travellers. 

image
I really should've finished this six months ago when I started.

### CODE REVIEW

**Comment by Stev-the-third** [comment]
!review

**Comment by Stev-the-third** [comment]
!wip

**Comment by Stev-the-third** [comment]
I forgot to add a changelog, I'll add one soon

**Comment by Stev-the-third** [comment]
!wip

**Comment by Stev-the-third** [comment]
!review

File: maps/away/away_site/automated_waystation/automated_waystation.dm [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+/area/automated_waystation/medical
+	name="Waystation Medical Module"
+	icon_state = "medbay"
+/area/automated_waystation/bathroom
+	name="Waystation Lavatory Module"
+	icon_state = "restrooms"
+/area/automated_waystation/bedroom
+	name="Waystation Rest and Recreation Module"
+	icon_state = "crew_quarters"
+/area/automated_waystation/canteen
```
Comment (DreamySkrell) [inline maps/away/away_site/automated_waystation/automated_waystation.dm:86]: needs newlines between area definitions

**Comment by DreamySkrell** [comment]
image
.dm file needs to be added to aurorastation.dme

**Comment by DreamySkrell** [comment]
image

scrubber pipe line should go to an empty canister, otherwise it's just going to go full quickly, and can't be 

supply pipe line here can't be refilled when emptied, needs either to just be an air canister, or keep the tank and add a canister port

**Comment by DreamySkrell** [comment]
image
needs free space navpoints that are not attached to any catwalks
rename the current navpoints to "catwalk dock" or something to differentiate them from free space navpoints

**Comment by DreamySkrell** [comment]
image
dark blue overmap icon is going to be hard to see on the overmap
make it like light blue or something

**Comment by DreamySkrell** [comment]
image
apc not connected to wires

**Comment by DreamySkrell** [comment]
image
use `/obj/machinery/shipsensors/strong/relay`

File: maps/away/away_site/automated_waystation/automated_waystation.dm [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	description = "Automated Waystation."
+	unit_test_groups = list(1)
+
+	id = "automated_waystation"
+	spawn_cost = 1
+	spawn_weight = 1
+	prefix = "away_site/automated_waystation/"
+	suffixes = list("automated_waystation.dmm")
+
+	sectors = list(ALL_COALITION_SECTORS)
```
Comment (DreamySkrell) [inline maps/away/away_site/automated_waystation/automated_waystation.dm:12]: this is effectively a different variant of the current sensor relay, and it is redundant if both spawn at the same time

so either:
- make this station guaranteed, and use the `ban_ruins` var on both to ban the other one (imo this is better, but make sure it actually works in game)
- make this station guaranteed, and blacklist the current sensor relay from spawning in the coc

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
as above

**Comment by Stev-the-third** [comment]
Closing this, since I want to prioritise other projects that will actually be seen within the next like year lmao. Will reopen whenever I get around to it.

**Pull request closed without merging**